recipes/master.rb

```diff
@@ -84,6 +80,16 @@
   end
 end
 
+ruby_block 'delay salt-master service start' do
+  block do
+  end
+  notifies :start, 'service[salt-master]'
```

Comment:
and there was some issue with kitchen test which got resolved by delaying the service start.

I do not remember what it was, but I can check it again if required.